### PR TITLE
OPENEUROPA-1569: Task-Runner: Add support in task-runner "process" task arrays (imploded to string with comma delimiter).

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

For example in `phpunit.xml.dist` we have this code:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit bootstrap="${drupal.root}/core/tests/bootstrap.php" backupGlobals="true" colors="true" >
  <php>
    <ini name="error_reporting" value="32767"/>
    <ini name="memory_limit" value="-1"/>
    <env name="SIMPLETEST_IGNORE_DIRECTORIES" value="build,vendor,node_modules,${drupal.root}"/>
    <env name="SIMPLETEST_BASE_URL" value="${drupal.base_url}"/>
    <env name="SIMPLETEST_DB" value="mysql://${drupal.database.user}:${drupal.database.password}@${drupal.database.host}:${drupal.database.port}/${drupal.database.name}"/>
  </php>
  <testsuites>
    <testsuite>
      <directory>./modules/**/tests/</directory>
      <directory>./tests/</directory>
    </testsuite>
  </testsuites>
</phpunit>
```

It would be nice if we could use in `SIMPLETEST_IGNORE_DIRECTORIES` the value `${drupal.settings.settings.file_scan_ignore_directories}`.

Update of `phpunit.xml.dist` file in all existing components is optional in terms of ticket's scope.

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: src/Traits/ConfigurationTokensTrait.php [hunk 1]
```diff
@@ -39,7 +39,11 @@ protected function extractProcessedTokens($text)
         $config = $this->getConfig();
 
         return array_map(function ($key) use ($config) {
-            return $config->get($key);
+            $value = $config->get($key);
+            if (is_array($value)) {
+                return implode(',', $value);
```
Comment (drupol) [inline src/Traits/ConfigurationTokensTrait.php]: What do we do if we have nested arrays ?

Comment (sergepavle) [reply src/Traits/ConfigurationTokensTrait.php]: reworked. Thanks!

**Comment by drupol** [comment]
You will need to wait for OPENEUROPA-1535 to be merged, then rebase, then move forward.

File: src/Commands/AbstractDrupalCommands.php [hunk 2]
```diff
@@ -391,7 +391,7 @@ public function permissionsSetup(array $options = [
 
         // Define collection of tasks.
         $collection = [
-            // Note that the chmod() method takes decimal values.
+        // Note that the chmod() method takes decimal values.
```
[the remark this answers is not in the document]
Comment (drupol) [reply src/Commands/AbstractDrupalCommands.php]: Fixed

**Pull request merged**